Python/YouTube_Trending_Analysis/YT_Trending_Analysis.py

```diff
@@ -0,0 +1,64 @@
+import json
```

Comment:
Final Checks
- add the updated output CSV file 
- readme update of Python folder

Reply:
Is this alright?